packages/visx-demo/package.json

```diff
@@ -98,7 +99,8 @@
     "react-markdown": "^4.3.1",
     "react-tilt": "^0.1.4",
     "recompose": "^0.26.0",
-    "topojson-client": "^3.0.0"
+    "topojson-client": "^3.0.0",
+    "webpack": "^5.75.0"
```

Comment:
when upgrading `next` I think we lost a transient `webpack` dependency that `happo` was accidentally depending on.

Reply:
not true, we actually configure happo to reference next's bundled version of webpack.